### ISSUE TITLE
 Qt Quick examples with pthreads doesn't work

### ISSUE THREAD

**Issue by weis** [msorvig/qt-webassembly-examples#14]
Has anyone brought a Qt Quick example to work with pthreads support?

While the Gui and Widgets examples work fine with pthreads support (standard settings, Google Chrome, Emscripten  V. 1.38.30), we always get the error message:
```
// at hellosquare.worker.js, line 144: 
var result = Module ['dynCall_ii'] (e.data.start_routine, e.data.arg);) 

TypeError: Can not read property 'devicePixelRatio' of undefined
      at __emval_get_property (Blob: http: // localhost / 0c566ee4-b3b8-4d12-beb2-486903d177aa: 1: 151478)
      at wasm-function [9286]: 34
      at wasm-function [17776]: 48
      at wasm-function [1384]: 32
      at wasm-function [2344]: 172
      at wasm-function [17795]: 1005
      at wasm-function [17796]: 317
      at wasm-function [23884]: 184
      at wasm-function [19850]: 13
      at Object.Module.dynCall_ii
(Blob: http: // localhost / 0c566ee4-b3b8-4d12-beb2-486903d177aa: 1: 340869)
onmessage @ hellosquare.worker.js: 179
```

The application seems to run into an  infinite loop.

**Comment by msorvig** [comment]
You need this patch: https://codereview.qt-project.org/#/c/256695/

Was merged to the 5.13 branch today, so you should get it if you pull now.

**Comment by weis** [comment]
Qt Quick demos are now running.
Thank you !!!
